[ews-build.webkit.org] Use new-style shell commands for services testing

JonWBedard opened the pull request:
#### 1a50a2d99c97a053efbf18744c7539b53ff04390
<pre>
[ews-build.webkit.org] Use new-style shell commands for services testing
<a href="https://bugs.webkit.org/show_bug.cgi?id=249029">https://bugs.webkit.org/show_bug.cgi?id=249029</a>
rdar://103186946

Reviewed by Aakash Jain.

* Tools/CISupport/ews-build/steps.py:
(RunWebKitPerlTests): Make step ShellCommandNewStyle.
(RunBuildWebKitOrgUnitTests): Ditto.
(RunEWSUnitTests): Ditto.
(RunResultsdbpyTests): Ditto.
(WebKitPyTest): Ditto.
(WebKitPyTest.run): Renmaed from WebKitPyTest.start.
(RunBuildWebKitOrgUnitTests.start): Deleted.
(WebKitPyTest.start): Renamed to WebKitPyTest.run.

Canonical link: <a href="https://commits.webkit.org/257767@main">https://commits.webkit.org/257767@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/38d0936d766c710806c09ee5cffb36e0c8d5fe91

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/6/builds/99563 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/77/builds/8738 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/43/builds/32656 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/8/builds/108925 "Built successfully") | [✅ 🛠 🧪 win](https://ews-build.webkit.org/#/builders/10/builds/169162 "Built successfully and passed tests") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/11/builds/103556 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/76/builds/9324 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/61/builds/86043 "Built successfully") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/36/builds/92037 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/12/builds/106842 "Built successfully") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/19/builds/105323 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/78/builds/7051 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/3/builds/90571 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/35/builds/34014 "Passed tests") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/9/builds/88867 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/73/builds/21923 "Passed tests") | [  ~~🧪 api-gtk~~](https://ews-build.webkit.org/#/builders/34/builds/77283 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/81/builds/2579 "Built successfully") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/70/builds/23438 "Passed tests") | | 
| [⏳ 🛠 🧪 jsc-arm64 ](https://ews-build.webkit.org/#/builders/JSC-Tests-arm64-EWS "Waiting in queue, processing has not started yet") | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/82/builds/2511 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/60/builds/45831 "Passed tests") | | 
| [✅ 🧪 services](https://ews-build.webkit.org/#/builders/20/builds/98493 "Passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/80/builds/8659 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/62/builds/42913 "Passed tests") | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/79/builds/4371 "Built successfully") | | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/75/builds/2742 "Built successfully and passed tests") | | | | 
<!--EWS-Status-Bubble-End-->

aj062 reviewed (approved):
r+ assuming this is tested (on uat instance or otherwise)

JonWBedard committed:
[ews-build.webkit.org] Use new-style shell commands for services testing
https://bugs.webkit.org/show_bug.cgi?id=249029
rdar://103186946

Reviewed by Aakash Jain.

* Tools/CISupport/ews-build/steps.py:
(RunWebKitPerlTests): Make step ShellCommandNewStyle.
(RunBuildWebKitOrgUnitTests): Ditto.
(RunEWSUnitTests): Ditto.
(RunResultsdbpyTests): Ditto.
(WebKitPyTest): Ditto.
(WebKitPyTest.run): Renmaed from WebKitPyTest.start.
(RunBuildWebKitOrgUnitTests.start): Deleted.
(WebKitPyTest.start): Renamed to WebKitPyTest.run.

Canonical link: https://commits.webkit.org/257767@main
